Sources/Components/HappinessRating/HappinessRating.swift

```diff
@@ -25,4 +25,12 @@ public enum HappinessRating: Int, CaseIterable {
             return UIImage(named: .ratingFaceLove)
         }
     }
+
+    func textLabel(text: String) -> Label {
```

Comment:
I think this method should rather be put in the view it's being used in, aka. the `HappinessRatingView`

Reply:
It was a property but I refactored and forgot to move it 😅  I assume you mean the `RatingImageView` class? if so then I agree.

The `HappinessRatingView` shouldn't need to know if the particular `ImageView` that is being added has a `TextLabel`.